backend/service/martini/fcc.go

```diff
@@ -66,36 +44,90 @@ func (s *Service) FccContactHandler() rest.Handle {
 			return
 		}
 
-		cached, err := s.db.GetFccContactResponse(r.Context(), frn)
-		if err == nil {
-			w.Header().Set(header.ContentType, header.ApplicationJSON)
-			w.Write([]byte(cached.Response))
+		res, err := s.getFccContact(r.Context(), frn)
+		if err != nil {
+			marshal.WriteJSON(w, r, httperror.WithUnexpected("request failed: "+err.Error()).WithCause(err))
 			return
 		}
 
-		fccClient := fcc.NewAPIClient(s.FccBaseURL)
-		cQueryResults, err := fccClient.GetContactResults(frn)
-		if err != nil {
-			marshal.WriteJSON(w, r, httperror.WithInvalidRequest("unable to get email response").WithCause(err))
-			return
+		logger.KV(xlog.DEBUG, "frn", frn, "response", res)
+		marshal.WriteJSON(w, r, res)
+	}
+}
+
+func (s *Service) getFrnResponse(ctx context.Context, filerID string) (*v1.FccFrnResponse, error) {
+	if filerID == "123456" {
```

Comment:
Do you intend to keep this? i understand you do not want to use a filer id for real company (i usually use filer id 831188, but make sure send email to contact is disabled when testing).

Reply:
`831188` is real number that belongs to Matthew.
He will need to issue real cert for his comany